azure-pipelines/vs-insertion.yml

```diff
@@ -0,0 +1,210 @@
+# Create a VS insertion (DotNet-MSBuild-Trusted -> VS) from a build artifact on main or any servicing branch. 
+trigger: none
+name: $(Date:yyyyMMdd).$(Rev:r)
```

Comment:
Is this overridden later? Ideally the name of the insertion job would relate to the version of MSBuild being inserted rather than the date of insertion.

Reply:
I don't think so. This is just migrated. I had an issue with changing it and interaction with CI; but I had a bad grasp of pipelines at that time, it can be improved. Seems low prio though - the inserted version is visible in the insertion PR description.